src/app/shared/data-storage.service.ts

```diff
@@ -13,7 +13,9 @@ import { AvailableSkill } from './availableSkill.model';
 export class DataStorage{
     teamsUrl = "http://localhost:3000/teams";
     usersUrl = "http://localhost:3000/users";
-    skillsUrl = "http://localhost:3000/skills";
+    genericSkillsUrl = "http://localhost:3000/genericskills";
+    domainSkillsUrl = "http://localhost:3000/domainskills";
+    kbcSkillsUrl = "http://localhost:3000/kbcskills";
```

Comment:
constants

Reply:
Please use routers for navigations to all those screens . it is not the right way to do it if this is declared for navigation

Reply:
done